### ISSUE TITLE
Upgrade from C131 -> C132 on Win x64 via 1.75.x - Nightly (Checking C132)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#42521]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [ ] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [ ] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [ ] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [ ] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [ ] `CertificateRevocation`
  - [ ] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [ ] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [ ] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [ ] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [ ] `Safe Browsing`
- [ ] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [ ] Make sure that data from the last version appears in the new version OK
- [ ] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [ ] With data from the last version, verify that:
  - [ ] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [ ] Cookies are preserved
  - [ ] Installed extensions are retained and work correctly
  - [ ] Opened tabs can be reloaded
  - [ ] Stored passwords are preserved
  - [ ] Sync chain created in previous version is retained
  - [ ] Social media-blocking buttons changes are retained
  - [ ] Custom filters under brave://settings/shields/filters are retained
  - [ ] Custom lists under brave://settings/shields/filters are retained
  - [ ] Rewards
    - [ ] BAT balance is retained
    - [ ] Auto-contribute list is retained
    - [ ] Both Tips and Monthly Contributions are retained
    - [ ] Panel transactions list is retained
    - [ ] Changes to rewards settings are retained
    - [ ] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [ ] Ads
    - [ ] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [ ] Changes to ads settings are retained
    - [ ] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [ ] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by kjozwiak** [comment]
Verified on `Win 10 x86` using the following build(s):

```
Brave | 1.75.48 Chromium: 132.0.6834.15 (Official Build) nightly (32-bit)
-- | --
Revision | 6875d37b04b8bc9c26dfb167ae8cb02ca945ab39
OS | Windows 10 Version 22H2 (Build 19045.5011
```

<details>
<summary>C131 --> C132 on-demand upgrade check - PASSED</summary>

* installed `1.75.36 Chromium: 131.0.6778.85` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates-panel.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.75.48 Chromium: 132.0.6834.15` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|---------------
![Image](https://github.com/user-attachments/assets/1150325a-ddb1-477d-9e13-440ca2bd0847) | ![Image](https://github.com/user-attachments/assets/57002b44-8762-42c7-bcd2-9760d713f965) | ![Image](https://github.com/user-attachments/assets/3c11d37d-32b1-492e-b8d8-5c8f59f54d20) | ![Image](https://github.com/user-attachments/assets/f45f7ed8-552d-4a50-8c4f-4121dedffb05)

**Comment by MadhaviSeelam** [comment]
Verification `In progress` using
```
Brave | 1.75.39 Chromium: 132.0.6834.15 (Official Build) nightly (64-bit)
-- | --
Revision | 4a2c1f978d65412aa0a7c6d2ad5c8f7a11b6a9af
OS | Windows 11 Version 24H2 (Build 26100.2314)
```

<details>
<summary>C131 --> C132 on-demand upgrade check - Pending</summary>

1. installed latest nightly `1.75.36 Chromium: 131.0.6778.85 (Official Build) nightly (64-bit)` and closed Brave
2. enable VPN if not enabled already
4. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:  * `https://updates-panel.brave.com/service/update2`
5. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
6. relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
7. ensured that clicking `Relaunch` restarted the browser without any issues
8. ensured Brave was updated to `1.75.39 Chromium: 132.0.6834.15 (Official Build) nightly (64-bit)` without any issues and you can restart the browser

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 6`**
----------------|----------------|----------------|---------------|------------
![Image](https://github.com/user-attachments/assets/2c8222ea-c1c6-406e-8646-5c7184267eed)
![Image](https://github.com/user-attachments/assets/2c16f2cb-b092-4dc7-beaf-8b1ad374c6cd)
![Image](https://github.com/user-attachments/assets/f9505b03-a7ee-4502-af62-f1aa61aa9d30)


</details>

<details>
<summary>C131 --> C132 background upgrade check - Pending</summary>

1. installed latest nightly `1.75.49 Chromium: 131.0.6778.85 (Official Build) nightly (64-bit) ` and closed Brave
2. enable VPN if not enabled already
3. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
-   * `https://updates-panel.brave.com/service/update2`
3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
5. deleted any `last checked` entry
6. started `BraveSoftwareUpdateTaskMachineUA{72133F36-BD88-4DB8-B8E4-DC9994873070}` via `Task Scheduler`
7. waited ~30 min or so before launching Brave and ensured it was updated to `1.75.39 Chromium: 132.0.6834.15 (Official Build) nightly (64-bit) `
8. restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`**
----------------|----------------|---------------|----------------|-----------
![Image](https://github.com/user-attachments/assets/0fdb4e19-ffeb-4c93-ba4f-4f139ea85e7f)
![Image](https://github.com/user-attachments/assets/ac164c49-b5a5-432d-95c5-e0664fc06c36)
![Image](https://github.com/user-attachments/assets/cf925249-7fcc-4e22-8658-903cefed5bba)
![Image](https://github.com/user-attachments/assets/d40f536a-4e97-4da0-8892-32b8030a55a3)



</details>

<details>
<summary>C132--> C132 on-demand upgrade check - Pending</summary>

1. installed `1.75.39 Chromium: 132.0.6834.15 (Official Build) nightly (64-bit) ` 
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
- * `https://updates.brave.com/service/update2`
3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
4. visited `brave://settings/help` and ensured that new update was being pulled/downloaded
5. ensured that clicking `Relaunch` restarted the browser without any issues
6. ensured Brave was updated to ` ` without any issues and you can restart the browser

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`**
----------------|----------------|----------------|---------------|----------


</details>

<details>
<summary>C131 --> C131  background upgrade check - Pending</summary>

1. installed `1.75.39 Chromium: 132.0.6834.15 (Official Build) nightly (64-bit) `  and closed Brave
2.  created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
    * `https://updates.brave.com/service/update2`
3.  set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
4. delete any `last checked` entry
5. started `BraveSoftwareUpdateTaskMachineUA{91C02688-57FE-475A-A4DC-781A543A330D}` via `Task Scheduler`
6. closed Brave
7. waited ~30min or so before launching Brave and ensured it was updated to ``
8. restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 7`**  
----------------|----------------|------------|------------|--------- 


</details>
